_docs/student/submission.md

```diff
@@ -39,7 +39,7 @@ Once you have selected all necessary files, press "Submit".
 Then wait patiently for your assignment to be graded.  The page will
 automatically refresh when grading is complete.  If the assignment is
 computationally intensive or if the server is busy grading your
-classmates work, you may need to wait longer.
+classmate's work, you may need to wait longer.
```

Comment:
let's make it `classmates' work` as there could be many classmates who all have their work being graded.